presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java

```diff
@@ -204,13 +204,13 @@ public void testInnerTpchQ7JoinOrder()
                         new Join(
                                 tpchSf10Table("lineitem"),
                                 new Join(
-                                        tpchSf10Table("orders"),
-                                        new Join(
-                                                tpchSf10Table("customer"),
-                                                tpchSf10Table("nation")))),
+                                        tpchSf10Table("supplier"),
```

Comment:
I observed the same thing when I rebased these commits onto statistics-13 (aka cbo-1)